service/service.go

```diff
@@ -404,9 +412,19 @@ func dropViewOption(selector *config.ViewSelector) config.View {
 func configureViews(level configtelemetry.Level) []config.View {
 	views := []config.View{}
 
-	if disableHighCardinalityMetricsFeatureGate.IsEnabled() {
-		views = append(views, []config.View{
-			{
+	if level < configtelemetry.LevelDetailed {
+		// Drop all otelhttp and otelgrpc metrics if the level is not detailed.
+		views = append(views,
+			dropViewOption(&config.ViewSelector{
+				MeterName: ptr("go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc"),
+			}),
+			dropViewOption(&config.ViewSelector{
+				MeterName: ptr("go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"),
+			}),
+		)
+	} else if disableHighCardinalityMetricsFeatureGate.IsEnabled() {
```

Comment:
Can you add a comment explaining the Go SDK behavior when there are multiple views that affect the same set of metrics?